apps/front-end/src/components/panel/directoryPanel/DirectoryPanel.tsx

```diff
@@ -40,17 +39,12 @@ const DirectoryPanel = () => {
 
   const directoryOptions = useAppSelector(selectDirectoryOptions);
   const activeValue = directoryOptions.value;
-
-  const [resultsTotals, setResultsTotals] = useState<number[]>([]);
+  const [resultsTotals, setResultsTotals] = useState<any>();
```

Comment:
If you use a switch statement above this doesn't need to be `<any>`, it can be something more appropriate like `Record<string, number>` (depending on what you return from `fetchResults`)